contents/blog/the-posthog-array-1-16-0.md

```diff
@@ -0,0 +1,202 @@
+---
+date: 2020-11-04
+title: Array 1.16.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+Following our largest release to date, we are now back on a more regular release schedule. And, given that scalability was the focus of the previous release, we thought it was time for some fresh new features. 
+
+So, without further ado, here they are:
+
+## Release Notes
+
+### [Session Recording (Beta)](https://github.com/PostHog/posthog/pulls?q=is%3Apr+session+recording+)
```

Comment:
Maybe reference the main PR or epic (https://github.com/PostHog/posthog/issues/1846)? Think it might be easier to understand what's going on.

Reply:
Referencing the issue is a good idea